<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsInfo</class>
 <widget class="QWidget" name="ContactsInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ContactsInfo</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Phone Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numLabel">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="callButton">
       <property name="text">
        <string>Call</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
